clients/src/test/java/org/apache/kafka/common/requests/ConsumerGroupRequestTest.java

```diff
@@ -28,40 +29,57 @@
 import static org.apache.kafka.common.requests.ConsumerGroupDescribeRequest.getErrorDescribedGroupList;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class ConsumerGroupDescribeRequestTest {
+public class ConsumerGroupRequestTest {
```

Comment:
We usually have a test class per class for unit tests. Let's revert this please and introduce ConsumerGroupHeartbeatRequestTest if it does not exist.

Reply:
oh I see, thank you for thie note